keyboards/xd75/keymaps/skewwhiffy/keymap.c

```diff
@@ -15,36 +15,44 @@
  */
 #include "xd75.h"
 #include "keymap_uk.h"
-// Fillers to make layering more clear
+
 #define _______ KC_TRNS
```

Comment:
This line is unnecessary and can be safely deleted, as QMK defines this automatically.

https://github.com/qmk/qmk_firmware/blob/27e216fc1a481bb3e65832458bd2b199cb499e0b/quantum/quantum_keycodes.h#L29-L31

Reply:
Done.